packages/backend/src/database/migrations/20231101164443_add_cache_explores_foreign_key.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from 'knex';
+
+const CACHED_EXPLORES_TABLE_NAME = 'cached_explores';
+
+export async function up(knex: Knex): Promise<void> {
```

Comment:
everything in this block runs as a transaction right?

Reply:
yes